<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoomLoginClass</class>
 <widget class="QWidget" name="ChatRoomLoginClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatRoomLogin</string>
  </property>
  <widget class="QCheckBox" name="m_chkAutoLogin">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>243</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(168, 168, 168);</string>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_chkRemPasswd">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>243</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(168, 168, 168);</string>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_labFindPasswd">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>243</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(168, 168, 168);</string>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnLogin">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>274</y>
     <width>236</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_labRegAccount">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>305</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(168, 168, 168);</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_labAnimation">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_labProfile">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>92</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-radius: 36px;
background-color: rgba(255, 255, 255,200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
